lib/send-mail.ts

```diff
@@ -117,7 +113,7 @@ export async function parseMBoxMessageIDAndReferences(mbox: string):
      */
     const msgIdRegex =
         /^\s*<([^>]+)>(\s*|,)(\([^")]*("[^"]*")?\)\s*|\([^)]*\)$)?(<.*)?$/;
-    for (const header of parsed.headers) {
+    for (const header of parsed.headers ?? []) {
```

Comment:
I guess `??` means that `[]` is substituted only if `parsed.headers` is `undefined`, while a single `?` would also substitute if `parsed.headers` is already the empty array?

Reply:
A single `?` sends the compiler into a conditional operator search state that errors out with `error TS1005: ':' expected.`.  The `??` gets past this.  Either I found an answer (SO?) or I just added `?` until the compiler was happy, knowing the only current caller of the code errors out if the headers are not found (as did the old code in this function).  The ts syntax is what it is and maybe using a local to reference the possibly not present array is cleaner.